### PR TITLE
Fix test runner

### PR DESCRIPTION
**Pull request by jtormey**

### CODE REVIEW

**Comment by jtormey** [comment]
CI passing again.

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ build: node_modules
 	npm run build
 
 test: build
```
Comment (Artic2019) [inline Makefile:9]: Да

File: karma.conf.js [hunk 2]
```diff
@@ -31,10 +31,10 @@ module.exports = function (config) {
         bundle.once('prebundle', function () {
           bundle.transform('browserify-istanbul'); // Must go first
           bundle.transform('babelify', {
-            presets: ['env'],
```
Comment (Artic2019) [inline karma.conf.js:34]: , ' es2015 '

File: karma.conf.js [hunk 3]
```diff
@@ -31,10 +31,10 @@ module.exports = function (config) {
...
+            presets: ['env', 'es2015'],
             ignore: [
               'src/ws-browser.js', // undefined is not an object (evaluating 'global.WebSocket')
-              /\/node_modules\/(?!bitcoin-(coinify|exchange|sfox)-client\/)/
```
Comment (Artic2019) [inline karma.conf.js:37]: | bech32

File: package.json [hunk 4]
```diff
@@ -74,7 +74,8 @@
     "babel-loader": "6.2.10",
     "babel-plugin-transform-object-assign": "~6.22.0",
     "babel-polyfill": "6.16.*",
-    "babel-preset-es2015": "~6.16.0",
```
Comment (Artic2019) [inline package.json:77]: 1

File: Makefile [hunk 5]
```diff
@@ -7,7 +7,7 @@ build: node_modules
...
-	# ./node_modules/karma/bin/karma start karma.conf.js --single-run
```
Comment (Artic2019) [inline Makefile:10]: Тест

File: yarn.lock [hunk 6]
```diff
@@ -540,7 +540,7 @@ babel-plugin-transform-async-generator-functions@^6.24.1:
     babel-plugin-syntax-async-generators "^6.5.0"
     babel-runtime "^6.22.0"
 
-babel-plugin-transform-async-to-generator@^6.24.1:
```
Comment (Artic2019) [inline yarn.lock:543]: 22.0, babel-plugin-transform-async-to-generator@^6.

File: yarn.lock [hunk 7]
```diff
@@ -594,7 +594,7 @@ babel-plugin-transform-es2015-block-scoped-functions@^6.22.0, babel-plugin-trans
   dependencies:
     babel-runtime "^6.22.0"
 
-babel-plugin-transform-es2015-block-scoping@^6.14.0, babel-plugin-transform-es2015-block-scoping@^6.24.1:
```
Comment (Artic2019) [inline yarn.lock:597]: 23.0, babel-plugin-transform-es2015-block-scoping@^6.

File: yarn.lock [hunk 8]
```diff
@@ -604,7 +604,7 @@ babel-plugin-transform-es2015-block-scoping@^6.14.0, babel-plugin-transform-es20
     babel-types "^6.26.0"
     lodash "^4.17.4"
```
Comment (Artic2019) [inline yarn.lock:606]: 23.0, babel-plugin-transform-es2015-block-scoping@^6.

**Review by West120912 — COMMENTED** [review]
Great

**Pull request merged**